storage/store.go

```diff
@@ -1280,11 +1283,16 @@ func raftEntryFormatter(data []byte) string {
 // updateStoreStatus updates the store's status proto.
 func (s *Store) updateStoreStatus() {
 	now := s.ctx.Clock.Now().WallTime
-	s.status.UpdatedAt = now
-	s.status.RangeCount = int32(len(s.ranges))
+	status := &proto.StoreStatus{
+		StoreID:    s.Ident.StoreID,
+		NodeID:     s.Ident.NodeID,
+		UpdatedAt:  now,
+		RangeCount: int32(len(s.ranges)),
+		Stats:      proto.MVCCStats(s.scanner.Stats().MVCC),
+	}
+	atomic.StorePointer(&s.status, unsafe.Pointer(status))
```

Comment:
I was hoping to store the started at time, which is when it was last started, not created.